mcs/class/System/Mono.AppleTls/Certificate.cs

```diff
@@ -309,6 +328,17 @@ public SecKey (IntPtr handle, bool owns = false)
 				CFObject.CFRetain (handle);
 		}
 
+		/*
```

Comment:
Hmm, we could also make `owner` the object that's `CFRelease()`d and set it to `IntPtr.Zero` if we don't "own" it.